install/installer/pkg/config/v1/config.go

```diff
@@ -49,6 +49,9 @@ func (v version) Defaults(in interface{}) error {
 	cfg.Metadata.Region = "local"
 	cfg.Metadata.InstallationShortname = "default" // TODO(gpl): we're tied to "default" here because that's what we put into static bridges in the past
 	cfg.ObjectStorage.InCluster = pointer.Bool(true)
+	cfg.ObjectStorage.Resources.Requests = corev1.ResourceList{
+		corev1.ResourceMemory: resource.MustParse("256Mi"),
```

Comment:
what was default prior to this change? or there was no requests at all?

Reply:
I'd prefer to have exactly the same behaviour as before, so if it wasn't set before I'd prefer not to set it by default now either ☺️

Reply:
Thanks for clarifying! Setting it to 2gb sounds good then ☺️